services/ux-backend/handlers/utils.go

```diff
@@ -0,0 +1,20 @@
+package handlers
+
+import (
+	"fmt"
+	"os"
+)
+
+var namespace string
+
+// returns namespace found in env value, will panic if value is empty
+func GetOperatorNamespace() string {
+	if namespace != "" {
+		return namespace
+	}
+	if ns := os.Getenv(podNamespaceEnvVar); ns != "" {
```

Comment:
This env var is only used once, in that case, can we please use a literal string instead?
It makes it easier to understand exactly what the code does without looking for the constant definition to read its value.

Reply:
yes, done.